include/wx/msw/webview_edge.h

```diff
@@ -0,0 +1,186 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        include/wx/msw/webview_edge.h
+// Purpose:     wxMSW Edge Chromium wxWebView backend
+// Author:      Markus Pingel
+// Created:     2019-12-15
+// Copyright:   (c) 2019 wxWidgets development team
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef wxWebViewEdge_H
+#define wxWebViewEdge_H
+
+#include "wx/setup.h"
+
+#if wxUSE_WEBVIEW && wxUSE_WEBVIEW_EDGE && defined(__WXMSW__)
+
+#include "wx/control.h"
+#include "wx/dynlib.h"
+#include "wx/webview.h"
+#include "wx/msw/private/comptr.h"
```

Comment:
Making `wxCOMPtr` public would probably be trivial. But it could also make sense to move the edge implementation to a private file so a using application wouldn't have it's include paths setup to include "Webview2.h".

Reply:
This is a good point that I didn't think about and would indeed be worth doing IMO.